src/query/functions-v2/tests/it/scalars/parser.rs

```diff
@@ -21,6 +21,7 @@ use common_ast::parser::token::Token;
 use common_ast::parser::tokenize_sql;
 use common_ast::Backtrace;
 use common_ast::Dialect;
+use common_datavalues::IntervalKind;
```

Comment:
can we move `IntervalKind` to `expression`? So that we can remove dependent on `datavalues`.

Reply:
We can do this. But we need make `ast::parser` use `expression`, too.

Reply:
IntervalKind should be defined in AST crate. 

And expression crate depends on AST crate.